apps/nestjs-api/src/reminder-emails/reminder-emails.controller.ts

```diff
@@ -126,33 +126,54 @@ export class ReminderEmailsController {
     }
 
     return {
-      message: `Follow-up reminder emails sent to ${
+      message: `Mentorship Follow-up Reminder Emails Sent: ${
         Object.keys(threeMonthsOldMentorshipMatches).length
-      } mentorship matches`,
+      }`,
     }
   }
 
-  // @Get('/s3cr3t-3ndp01nt-t0-tr1gg3r-r3m1nd3r5/mentees-platform-and-new-mentors')
-  // async sendUnmatchedMenteesReminder() {
-  //   const unmatchedMenteesWithApprovedProfiles =
-  //     await this.reminderEmailsService.getUnmatchedMenteesWithApprovedProfiles()
+  @Get('/s3cr3t-3ndp01nt-t0-tr1gg3r-r3m1nd3r5/mentees-platform-and-new-mentors')
+  async sendUnmatchedMenteesReminder() {
+    const unmatchedMenteesFor45Days =
+      await this.reminderEmailsService.getUnmatchedMenteesFor45Days()
 
-  //   if (unmatchedMenteesWithApprovedProfiles.length > 0) {
-  //     // send reminder emails
-  //     unmatchedMenteesWithApprovedProfiles.forEach(async (mentee) => {
-  //       await this.reminderEmailsService.sendMenteesPlatformAndNewMentorsReminder(
-  //         {
-  //           email: mentee.props.email,
-  //           firstName: mentee.props.firstName,
-  //         }
-  //       )
-  //     })
-  //   }
+    if (Object.keys(unmatchedMenteesFor45Days).length > 0) {
+      unmatchedMenteesFor45Days.forEach(async (mentee) => {
+        // Send reminder email to mentee
+        await this.reminderEmailsService.sendMenteesPlatformAndNewMentorsReminder(
+          {
+            email: mentee.props.email,
+            firstName: mentee.props.firstName,
+          }
+        )
+      })
+    }
 
-  //   return {
-  //     message: `Reminder emails sent to ${unmatchedMenteesWithApprovedProfiles.length} unmatched mentees with approved profiles`,
-  //   }
-  // }
+    return {
+      message: `Platform Update Reminder Emails Sent to Mentees: ${unmatchedMenteesFor45Days.length}`,
+    }
+  }
+
+  @Get('/s3cr3t-3ndp01nt-t0-tr1gg3r-r3m1nd3r5/pending-mentorships')
+  async sendPendingMentorshipsReminder() {
+    const pendingMentorshipMatches =
+      await this.reminderEmailsService.getPendingMentorshipMatches()
+
+    if (Object.keys(pendingMentorshipMatches).length > 0) {
+      Object.keys(pendingMentorshipMatches).forEach(async (match) => {
+        // Send reminder email to mentee
+        await this.reminderEmailsService.sendMentorPendingApplicationReminder({
+          match: pendingMentorshipMatches[match],
+        })
+      })
+    }
+
+    return {
+      message: `Pending Mentorship Reminder emails sent to Mentors: ${
+        Object.keys(pendingMentorshipMatches).length
```

Comment:
And here `Pending Mentorship Reminder...` to `Pending Mentee Application Reminder...`.

Reply:
Fixed!